setup.py

```diff
@@ -13,10 +13,12 @@
       classifiers=[
           'License :: OSI Approved :: MIT License',
           'Programming Language :: Python :: 3',
+          'Programming Language :: Python :: 3 :: Only',
```

Comment:
And what when Python 4 is released? =) I suggest removing this change.

Reply:
then i'll just remove the python3 as well...  if you can't maintain a classifier for a hypothetical but unexpected python major release that has been on average every decade or so so far...